test/testinput/bufr_ncep_snow_adpsfc.yaml

```diff
@@ -39,10 +39,15 @@ observations:
 
           dewpointTemperature:
             query: "*/TMDP"
-        
+
           totalSnowDepth:
             query: "[*/SNWSQ1/TOSD, */MTRMSC/TOSD, */STGDSNDM/TOSD]"
-
+            transforms:
+              - scale: 1000.0
```

Comment:
@YoulongXia-NOAA this file was given to me I thought as an update

however with the changes this corresponds to the unit change to 'mm' reverting this but do like the upperBound will scale it into meters